lib/scope.js

```diff
@@ -671,6 +671,7 @@ AST_Toplevel.DEFMETHOD("mangle_names", function(options) {
             && node instanceof AST_VarDef
             && node.value instanceof AST_Lambda
             && !node.value.name
+            && keep_name(options.keep_fnames, node.name.definition().name)
```

Comment:
`node.name.name` is a bit faster and shorter too!

Reply:
updated